docs/plugins/spotify.rst

```diff
@@ -37,6 +58,24 @@ Command-line options include:
 * ``--show-failures`` or ``-f``: List the tracks that did not match a Spotify
   ID.
 
+A Spotify ID or URL may also be provided to the ``Enter release ID:``
+prompt during import::
```

Comment:
Here's another opportunity for active voice: for example, "You can enter the URL for an album or song on Spotify at the `enter Id` prompt during import."